### PR TITLE
docs: more tweaks so everything works on RTD

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -18,36 +18,57 @@ ops.charm module
 ----------------
 
 .. automodule:: ops.charm
+   :members:
```
Comment (jameinel) [inline docs/index.rst:21]: seems unfortunate to have to repeat yourself for each one vs inheriting from a common base.

Comment (chipaca) [reply docs/index.rst:21]: I agree. I'm going to see if I can back out the change -- I suspect the actual error was masking the problem.

**Comment by chipaca** [comment]
Confirmed the changes to conf.py and index.rst weren't necessary; the problem was dependencies all along :-/

File: docs/conf.py [hunk 2]
```diff
@@ -81,3 +81,9 @@
 #              concatenated and inserted.
 #     'init' - Only the __init__ method’s docstring is inserted.
 autoclass_content = 'both'
+
+autodoc_default_options = {
+    'members': None,            # None here means "yes"
+    'undoc-members': None,
+    'show-inheritance': None,
+}
```
Comment (jameinel) [inline docs/conf.py]: :+1:

**Pull request merged**